### ISSUE TITLE
Async pi example - HandleScope?

### ISSUE THREAD

**Issue by bazineta** [nodejs/nan#393]
In the async pi example, HandleScope is used as follows:
```
  // Executed when the async work is complete
  // this function will be run inside the main event loop
  // so it is safe to use V8 again
  void HandleOKCallback () {
    HandleScope();

    Local<Value> argv[] = {
        Null()
      , New<Number>(estimate)
    };

    callback->Call(2, argv);
  }
```

Is this a correct usage of HandleScope, or should the declaration instead be:

```
HandleScope scope;
```

**Comment by kkoopa** [comment]
No, it is incorrect usage. The latter is correct.

**Comment by kkoopa** [comment]
Fixed 3ebc23e73629f83b83241f781e253fad96bac215
